File: fix_strings.py
import re
with open(r'c:\Users\MVP\Downloads\PeerPal\Peerpal\src\pages\dashboard\SessionDetail.jsx', 'r', encoding='utf-8') as f:
    text = f.read()

text = text.replace('comments: Knowledge: , Comm:', 'comments: \Knowledge: \, Comm: \\')

with open(r'c:\Users\MVP\Downloads\PeerPal\Peerpal\src\pages\dashboard\SessionDetail.jsx', 'w', encoding='utf-8') as f:
    f.write(text)

File: fix_strings.js
@
const fs = require("fs");
const path = "C:/Users/MVP/Downloads/PeerPal/Peerpal/src/pages/dashboard/SessionDetail.jsx";
let t = fs.readFileSync(path, "utf8");
t = t.replace("comments: Knowledge: , Comm:", "comments: `Knowledge: ${ratings.knowledge || 0}, Comm: ${ratings.communication || 0}`");
fs.writeFileSync(path, t);
@
